src/components/common/CreateUserWizard/CreateUserWizardValidation.tsx

```diff
@@ -0,0 +1,100 @@
+import { string, object, array, ValidationError } from 'yup';
+
+import {
+  getUserByEmail,
+  GetUserByEmailQuery,
+  GetUserByEmailQueryVariables,
+  getUserByName,
+  GetUserByNameQuery,
+  GetUserByNameQueryVariables,
+} from '~gql';
+import { intl } from '~utils/intl';
+import { runQuery } from '~utils/yup';
+
+/*
+ * The username regex is composed of
+ * ^[A-Za-z0-9] starts with upper case, lower case or numerals
+ * [A-Za-z0-9-_.] can include upper case, lower case, numerals, underscore, hyphen or period
+ * {0,99}$ match the preceding set at least 0 and at most 99 times from the end (so excluding the first char)
+ */
+const USERNAME_REGEX = '^[A-Za-z0-9][A-Za-z0-9-_.]{0,99}$';
```

Comment:
In the dapp, the validation message for usernames is: "'Only characters a-z, 0-9, - and . are allowed'". I have followed this, but included underscores like in the colony name.

Reply:
I guess this should be in line with colonies and be up to 255 chars